src/SWP/Component/MultiTenancy/Repository/TenantRepositoryInterface.php

```diff
@@ -26,12 +26,24 @@
      * Finds the tenant by subdomain.
      *
      * @param string $subdomain The subdomain
+     * @param string $domain    The domain
      *
      * @throws \Doctrine\ORM\NonUniqueResultException
      *
      * @return TenantInterface|null The instance of TenantInterface or null
      */
-    public function findOneBySubdomain($subdomain);
+    public function findOneBySubdomainAndDomain($subdomain, $domain);
+
+    /**
+     * Finds the tenant by domain.
+     *
+     * @param string $domain The domain
+     *
+     * @throws \Doctrine\ORM\NonUniqueResultException
```

Comment:
Ok ;) But You added it originally - https://github.com/superdesk/web-publisher/commit/cc17be0f80f16d654f0458a73cc6d2ce86f4d955#diff-6d78e0c57d88d61b951b127912ddc261R30